src/util/debug.c

```diff
@@ -305,8 +297,8 @@ void sss_vdebug_fn(const char *file,
     }
 #endif
 
-    if (debug_timestamps) {
-        if (debug_microseconds) {
+    if (debug_timestamps == 1) {
```

Comment:
I think some define may be preferred here instead of just `1`.
Maybe expand defines from `debug.h`:
```
#define SSSDBG_TIMESTAMP_UNRESOLVED      -1
#define SSSDBG_TIMESTAMP_DEFAULT          1

#define SSSDBG_MICROSECONDS_UNRESOLVED   -1
#define SSSDBG_MICROSECONDS_DEFAULT       0
```